test/dynamo/test_higher_order_ops.py

```diff
@@ -867,14 +870,14 @@ def fn(*, x, y, z=None):
         counters.clear()
         opt = torch.compile(f, backend="eager", fullgraph=True)
         opt(x, y)
-        self.assertEqual(counters["stats"]["calls_captured"], 1)
+        self.assertEqual(counters["stats"]["calls_captured"], 2)
```

Comment:
To check my understanding: this changed from 1->2 because we now have a call turning the output into a Tuple?

Reply:
See [this comment](https://github.com/pytorch/pytorch/pull/109433#discussion_r1337506405).